kivy/uix/textinput.py

```diff
@@ -454,7 +454,7 @@ class TextInput(FocusBehavior, Widget):
                   'on_quad_touch')
 
     def __init__(self, **kwargs):
-        self.always_release_bubble = kwargs.get('always_release_bubble', False)
+        self.always_release_bubble = kwargs.get('always_release_bubble', True)
```

Comment:
I think the keyword argument is not strictly needed to fix the bug you were trying to fix. That's a different feature and customization option. What i was saying is, you should not care about the value of always_release, and most importantly don't change the default value yourself. I'm going to close this pr because it's a collective waste of time for everyone involved, please open a new pr if you have a fix that doesn't merely hide the bug.